### PR TITLE
Fix ipv6 datagram tunnel

### LINKED ISSUE #1619: Tunnel community address is None in encode_address()
This is on windows 8.1

```
ERROR   1439993592.07             log:443   Unhandled Error
Traceback (most recent call last):
  File "twisted\python\log.pyo", line 73, in callWithContext
    
  File "twisted\python\context.pyo", line 118, in callWithContext
    
  File "twisted\python\context.pyo", line 81, in callWithContext
    
  File "twisted\internet\selectreactor.pyo", line 151, in _doReadOrWrite
    
--- <exception caught here> ---
  File "twisted\internet\udp.pyo", line 234, in doRead
    
  File "Tribler\community\tunnel\Socks5\server.pyo", line 64, in datagramReceived
    
  File "Tribler\community\tunnel\routing.pyo", line 98, in tunnel_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1178, in tunnel_data_to_end
    
  File "Tribler\community\tunnel\conversion.pyo", line 473, in encode_data
    
  File "Tribler\community\tunnel\conversion.pyo", line 463, in encode_address
    
exceptions.TypeError: must be string, not None
```

### PR DESCRIPTION
**Pull request by whirm**
Closes #1619

### CODE REVIEW

File: Tribler/community/tunnel/Socks5/conversion.py [hunk 1]
```diff
@@ -143,8 +143,8 @@ def __decode_address(address_type, offset, data):
         offset += 1
         destination_address = data[offset:offset + domain_length]
         offset += domain_length
-    elif address_type == ADDRESS_TYP_IPV6:
-        return offset, None
+    elif address_type == ADDRESS_TYPE_IPV6:
+        IPV6AddrError()
```
Comment (LipuFei) [inline Tribler/community/tunnel/Socks5/conversion.py]: should it be `raise IPV6AddrError()`?

File: Tribler/community/tunnel/Socks5/conversion.py [hunk 2]
```diff
@@ -249,3 +249,8 @@ def encode_udp_packet(rsv, frag, address_type, address, port, payload):
     ]
 
     return ''.join(strings)
+
+
+class IPV6AddrError(NotImplementedError):
+    def __str__(self):
+        return "IPV6 support not implemened"
```
Comment (LipuFei) [inline Tribler/community/tunnel/Socks5/conversion.py]: typo: "implemented", missing a "t".

**Comment by whirm** [comment]
retest this please

**Comment by whirm** [comment]
I'll try a few times more

**Comment by whirm** [comment]
retest this please

**Comment by whirm** [comment]
@LipuFei ready

**Pull request merged**